core/src/main/java/org/elasticsearch/common/geo/GeoUtils.java

```diff
@@ -419,6 +428,8 @@ public static GeoPoint parseGeoPoint(XContentParser parser, GeoPoint point) thro
                 } else {
                     return point.resetFromGeoHash(geohash);
                 }
+            } else if (numberFormatException != null) {
+                throw new ElasticsearchParseException("[{}] and [{}] must be valid double values", numberFormatException, LATITUDE, LONGITUDE);
```

Comment:
This line will break our `precommit` checks because it violates the 140-character line-length limit.